executor/import_into.go

```diff
@@ -159,5 +168,21 @@ func (e *ImportIntoExec) doImport(distImporter *loaddata.DistImporter, task *pro
 		userStmtCtx.SetMessage(importResult.Msg)
 		userStmtCtx.SetAffectedRows(importResult.Affected)
 	}
+	if err2 := flushStats(ctx, se, e.importPlan.TableInfo.ID, distImporter.Result(task)); err2 != nil {
+		logutil.BgLogger().Error("flush stats failed", zap.Error(err2))
+	}
 	return err
 }
+
+// flushStats flushes the stats of the table.
+func flushStats(ctx context.Context, se sessionctx.Context, tableID int64, result importer.JobImportResult) error {
```

Comment:
we can add GetStatsCollector to session and use statsCollector.Update? that's more straightforward

Reply:
don't want to deal with import circle... https://github.com/pingcap/tidb/blob/f098548642c3f05d6e11c54bd3012b9f58cfb3a5/sessionctx/context.go#L129